packages/app/config/jest/setupTests.js

```diff
@@ -26,4 +29,13 @@ jest.mock("workers", () => {
   return {
     getOpScriptWorker: () => {}
   };
-});
\ No newline at end of file
+});
+
+jest.setTimeout(15000)
+configure({
+  asyncUtilTimeout: 15000
+})
+nock.disableNetConnect()
+nock.enableNetConnect(
+  host => host.includes('127.0.0.1') || host.includes('localhost')
+)
```

Comment:
Here we prevent unit tests calling actual endpoints - unless they're hosted locally (endpoints that are usually mocked in tests).

Reply:
Please add that as a comment in the code.

Reply:
Yes please @MoSattler - great suggestion @sreis